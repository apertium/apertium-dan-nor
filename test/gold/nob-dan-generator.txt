[-V0vIHPaFb4P]
Angst er en følelse af uro, anspændthed og nagende forventning om, at noget farligt vil ske, eller en overdreven frygtreaktion på en hændelse. [/option]
[/-V0vIHPaFb4P]
[-fdqv8uMuzLN]
Biopsi er en vævsprøve, der er fjernet fra en levende organisme for at undersøge, om det foreligger sygelige forandringer. Hensigten med biopsier er at præcisere, om der er behov for yderligere undersøgelser og behandling. [/option]
[/-fdqv8uMuzLN]
[0AQS_LbbTu4t]
det er en rovfugl, der er udbredt [/option]
[/0AQS_LbbTu4t]
[0fNOhzUY_A9i]
Karsygdomme er en fællesbetegnelse for sygdomme i blodårerne, både arterier og vener. [/option]
[/0fNOhzUY_A9i]
[0hALHGLyo0Bi]
Giveren må ikke have sygdom, der kan overføres gennem blod (for eksempel syfilis, malaria, hepatitis, HIV og andre vira eller bakterielle infektioner) eller have været akut syg de sidste to uger før tapning. [/option]
[/0hALHGLyo0Bi]
[1x4sVzQ8s9s1]
Om sommeren er der mange danskere, der cykler. Også selv om det regner. [/option]
Om sommeren er der mange danskere, der cykler. Også selvom det regner. [/option]
[/1x4sVzQ8s9s1]
[2PNxxSpAa4MU]
et karakteristisk ravinelandskab [/option]
[/2PNxxSpAa4MU]
[2wwB9ZRC1rOC]
Flere EU-lande med grænse til Rusland kan komme til at forbyde russiske turister at komme ind i landet, hvis EU ikke kommer med et generelt forbud mod turister fra Rusland. [/option]
[/2wwB9ZRC1rOC]
[3MMHomQNjk0f]
Det lykkedes imidlertid de to lande at undgå en handelskrig. [/option]
[/3MMHomQNjk0f]
[3sk50h4xINoB]
Riget grundlagdes sandsynligvis midt i 800-tallet af svenske vikinger, der underlagde sig området syd for Ladoga, og derefter udvidede herredømmet i 880'erne til også at omfatte Kiev-området. [/option]
[/3sk50h4xINoB]
[4pY9KoYRNWrk]
For at sikre nok nye røde blodceller efter tapning af 450 ml blod anbefales det at tappe blod kun 4 gange årligt. [/option]
For at sikre nok nye røde blodsceller efter tapning af 450 ml blod anbefales det at tappe blod kun 4 gange årligt. [/option]
[/4pY9KoYRNWrk]
[57KhqMt0wmzr]
på den anden side [/option]
[/57KhqMt0wmzr]
[5pP8Q6Gc8x6t]
Karsygdomme omfatter sygdomprocesser som årebetændelser (arteritis, flebitis), aterosklerose og blodprop, sjældnere også svulster (angiom). [/option]
[/5pP8Q6Gc8x6t]
[6COebhF1E2Wb]
Titanic sank på sin jomfrurejse til trods for, at det var erklæret synkefrit [/option]
[/6COebhF1E2Wb]
[6KvLKp1kq_xT]
Der vises film. [/option]
[/6KvLKp1kq_xT]
[6w_Cyw_s5ZbW]
et tysk modsvar [/option]
[/6w_Cyw_s5ZbW]
[9WnIBotlzcPP]
Typisk vil man inden for moderne personlighedspsykologisk forskning derfor have en trækpsykologisk tilnærmelse. [/option]
[/9WnIBotlzcPP]
[AS6H6I1OmsgK]
Destillation af aromatiske olier fra planter er beskrevet både i gamle kinesiske og egyptiske skrifter. [/option]
[/AS6H6I1OmsgK]
[B1eOkQOxw08p]
Det starter et nyt kursus om 14 dage. [/option]
[/B1eOkQOxw08p]
[B6JaEn8qY4lU]
For at komme udfordringerne i møde, ønsker nu regeringen at ændre plan- og bygningsloven. Kort forklaret betyder det, at grundejere skal tage langt mere ansvar for at forhindre vandrelaterede skader. [/option]
[/B6JaEn8qY4lU]
[Bf1sj1J3_fgB]
den udenrigs- og sikkerhedspolitiske justering [/option]
den udenrigs- og sikkerhedspolitiske justeringen [/option]
[/Bf1sj1J3_fgB]
[CPqqMmRGmvPQ]
Sanseceller skal imidlertid også reagere på andre (inadekvate) stimuli, men deres intensitet skal være meget kraftigere for at aktivere receptorerne. [/option]
[/CPqqMmRGmvPQ]
[DErPv2VMubRs]
hvilke byer [/option]
[/DErPv2VMubRs]
[DWfCnYS0Za9R]
Det er kun læger og psykologer, der kan være fagligt ansvarlige for udredning og behandling af psykiske lidelser. [/option]
[/DWfCnYS0Za9R]
[E69xPcv3u6jF]
Jeg synes nu at det er helt *okay. Det er nogle gange hyggeligt at det bare er os fra kontoret. [/option]
Jeg synes nu at det er helt okay. Det er nogle gange hyggeligt, at det kun er os fra kontoret. [/option]
[/E69xPcv3u6jF]
[ELNDjrMeoXml]
Fra det frosne væv kan der skæres tynde snit, der farves, sådan at man i løbet af omkring en halv time kan stille en foreløbig diagnose. [/option]
[/ELNDjrMeoXml]
[HgPh8_CcygPm]
et iPhone-problem [/option]
[/HgPh8_CcygPm]
[I7NRhBsQ1x4H]
en lærestol [/option]
[/I7NRhBsQ1x4H]
[JOXzDXbN6Lo_]
en relativt fin lærebog [/option]
[/JOXzDXbN6Lo_]
[JiUYj3lcaLUI]
Psykiske lidelser kan også være et resultat af fysiske og psykiske belastninger efter anden sygdom. [/option]
[/JiUYj3lcaLUI]
[Jke5Xn5HiMU9]
Det regner. [/option]
[/Jke5Xn5HiMU9]
[LFj9PnMAN0m9]
Karsygdomme giver derved ophav til nogle af de almindeligste og alvorligste sygdomstilstande som aorta-aneurisme, slagtilfælde, hjerteinfarkt og lungeembolisme. [/option]
[/LFj9PnMAN0m9]
[La9HqYX4OTl8]
Hvad sker med os og vores samfund, dersom vi tror på en indiskutabel åbenbaring og slutter at tænke? [/option]
[/La9HqYX4OTl8]
[LyEBEtFCbGKr]
hvilken by [/option]
[/LyEBEtFCbGKr]
[Mv6OMsSDl14k]
fredsopgørene [/option]
[/Mv6OMsSDl14k]
[NIVpTnBH7NKG]
en senere omdiskuteret stopordre [/option]
[/NIVpTnBH7NKG]
[OTvOC_guDR7P]
med de samme egenskaber som menneskerne [/option]
[/OTvOC_guDR7P]
[PDeZWDuecO4P]
og hendes bedste ven hedder Tommy [/option]
[/PDeZWDuecO4P]
[PgQbI5sKvvOj]
Ja, det gør jeg helt sikkert, men det er virkelig surt, at der ikke må komme kærester med. [/option]
[/PgQbI5sKvvOj]
[PymI-iPap2RL]
Hovedsymptomet er angst i situationer, hvor personen er udsat for ukendte mennesker eller for mulig granskning af andre. Personen er bange, for at han skal optræde på en måde, der vil være nedværdigende eller pinlig. [/option]
[/PymI-iPap2RL]
[QJzHLHp6zIUp]
de allierede [/option]
[/QJzHLHp6zIUp]
[QgTZsqSLjwjS]
Inden for idrætsmedicinen lægges der imidlertid betydelig vægt på regelmæssig vejledning under træning og konkurrencer. [/option]
[/QgTZsqSLjwjS]
[R0NfHR7ZfjsB]
Oslo er ikke planlagt for fremtidens styrtnedbør, heller tvært imod. [/option]
[/R0NfHR7ZfjsB]
[RMBRkDyMsyJJ]
personerne dræbtes umiddelbart [/option]
[/RMBRkDyMsyJJ]
[RyVFEsQt_nVl]
hun kan ikke lide dette noget særligt [/option]
[/RyVFEsQt_nVl]
[T6rtDLF3hR86]
Det gør ondt. [/option]
[/T6rtDLF3hR86]
[TNO6vILtLM_q]
Det lykkedes imidlertid Luther at flygte. [/option]
[/TNO6vILtLM_q]
[U-lthZ4hNtMU]
i såkaldt fri og grundlæggende forskning [/option]
[/U-lthZ4hNtMU]
[UE8i8Dh7zTWM]
Jeg har åle i min luftpudebåd [/option]
[/UE8i8Dh7zTWM]
[VoUl2ZxdUTrn]
på samme tid [/option]
[/VoUl2ZxdUTrn]
[XrGqkvRZ6txF]
den humanistiske klasse [/option]
[/XrGqkvRZ6txF]
[YvtYod61AUfZ]
Desuden er der sygdomstilstande, der fører til specielle sygelige (patologiske) smerteformer. [/option]
[/YvtYod61AUfZ]
[ZjlT_hks49HT]
tyske styrker [/option]
[/ZjlT_hks49HT]
[_2zJOIiBGsSB]
en senere omdiskuteret ordre [/option]
[/_2zJOIiBGsSB]
[_92F_iHPtNNU]
dræbtes umiddelbart [/option]
[/_92F_iHPtNNU]
[_F1WITBIh8rh]
Det lykkedes imidlertid ikke flere grundige studier at påvise sundhedseffekter, og efter 2. Verdenskrig blev magnetisme forladt som helbredelsesmetode inden for den videnskabsbaserede medicin. [/option]
[/_F1WITBIh8rh]
[_TUs_iEeAdzD]
til VG kl. 19 [/option]
[/_TUs_iEeAdzD]
[aFW9hQOm7pTE]
I en situation med behov for førstehjælp skal sikkerheden først være varetaget for førstehjælperen. [/option]
[/aFW9hQOm7pTE]
[a_DwkLG4H_di]
Det er nu på fredag, der er sommerfest [/option]
[/a_DwkLG4H_di]
[bExjU9XD6xCU]
ca. 400 e.Kr. [/option]
[/bExjU9XD6xCU]
[bLeZbeB0uX5J]
den uafhængige stat [/option]
[/bLeZbeB0uX5J]
[bZOK2GBR97Z_]
Stortinget bestemte. [/option]
[/bZOK2GBR97Z_]
[bnXuLAAvkoLr]
Det lykkedes imidlertid ikke Oppositionspartierne at samle sig om nogen hastevedtagelse. [/option]
[/bnXuLAAvkoLr]
[bopu512Hva28]
Statens havarikommission har besluttet at iværksætte en fuld sikkerhedsundersøgelse, efter at Tretten bro kollapsede mandag 15. august. [/option]
[/bopu512Hva28]
[dKxf-_irk_qd]
Der starter et nyt kursus om 14 dage. [/option]
[/dKxf-_irk_qd]
[e7whSWefDp2f]
den direkte pressestøtte [/option]
[/e7whSWefDp2f]
[eZgtN-3HgEiy]
Personlighed er defineret som de relativt stabile individuelle forskelle i, hvordan forskellige mennesker tænker, handler og føler på tværs af forskellige situationer. [/option]
[/eZgtN-3HgEiy]
[e__pH5O_IXrG]
Det banker på døren. Hvem tror du, at det er? [/option]
[/e__pH5O_IXrG]
[edeETerdHFNC]
Planteolierne er vældig koncentrerede, og anses af udøvere af aromaterapi for at være virksomme selv i små mængder. [/option]
[/edeETerdHFNC]
[fbSfW4Rc22od]
I selvbiografien skriver Rushdie at han aldrig havde hørt ordet fatwa før, og at ham ikke i sin vildeste fantasi havde påtænkt at en bogudgivelse villes føre til optøjer. [/option]
[/fbSfW4Rc22od]
[lI7hQm9NW8av]
Det er dumt at køre for stærkt. [/option]
Det er dumt, at køre for stærkt. [/option]
[/lI7hQm9NW8av]
[lSDVPRGrCbBZ]
På sin specielle måde skriver han om grundlaget for den fundamentalisme, som nylig holdt på at koste ham livet: [/option]
[/lSDVPRGrCbBZ]
[liZAWyCanT6V]
Karsygdomme kan medføre både udvidelser (aneurismer, åreknuder) og forsnævringer og tilstopninger af blodårerne. [/option]
[/liZAWyCanT6V]
[lmZbhKy_sW8Q]
den nye colombianske stat [/option]
[/lmZbhKy_sW8Q]
[lpaouKQGB1Fz]
Nu skal du tage opvasken [/option]
[/lpaouKQGB1Fz]
[mgR6gyAKGX8l]
Vi er ikke tilfredse med de tiltag, regeringen skitserer. [/option]
[/mgR6gyAKGX8l]
[n98k8JcSizvA]
Man har tidligere været helt ubevist for vandproblematik og klimaændringer, når man har bygget, siger hun. [/option]
[/n98k8JcSizvA]
[nK5v-QUB-q8b]
Ca. 80–90 procenter af patienterne bliver raske. [/option]
[/nK5v-QUB-q8b]
[oCrYI0MpnnGh]
Det betyder, at mindst fire patienter skal behandles, for at en ekstra patient skal blive frisk. [/option]
[/oCrYI0MpnnGh]
[oIoJa8_20L1K]
Jeg synes nu, at det er helt okay. Det er nogle gange rart, at det bare er os fra kontoret. [/option]
[/oIoJa8_20L1K]
[oaKVcPJD3xPr]
Ifølge Solberg skal offentlige myndigheder kortlægge hvilke af sine tage, der egner sig for solceller. Staten skal gå foran på egne byger, mens Enova skal få i opgave at udforme en tidsbegrænset støtteordning, der er enkel og ubyråkratisk for næringsbyg. [/option]
[/oaKVcPJD3xPr]
[pyo8SJjgCh1d]
Gardariget er i nordisk sagalitteratur navnet på Kievriget, et rige i det nuværende Rusland og Ukraine, der lå omkring Novgorod (på norrønt Holmgarðr) og Kiev. [/option]
[/pyo8SJjgCh1d]
[qbLx1wio3URc]
Havarikommissionen skal undersøge kollapset af Tretten bro. [/option]
[/qbLx1wio3URc]
[soZfxNO1iCxq]
Der er nogen, der banker på døren. [/option]
[/soZfxNO1iCxq]
[tNaIcvfkZQ88]
Jeg tror, der kommer mange i år. [/option]
[/tNaIcvfkZQ88]
[unXNNvEy3DVm]
en manglende vilje [/option]
[/unXNNvEy3DVm]
[uyaRYR-6aara]
Finder man en bevidstløs person, skal man tjekke, om vedkommende trækker vejret. [/option]
[/uyaRYR-6aara]
[wBwWxw5blUow]
En tysk-sovjetisk *ukjentordher. [/option]
[/wBwWxw5blUow]
[wMR3JZqZ2bQp]
I praksis har de retslærde [/option]
[/wMR3JZqZ2bQp]
[wYh4HfrgvFOJ]
Wikipedia er en international internetbaseret encyklopædi der udgives af den ideelle organisation Wikimedia Foundation [/option]
Wikipedia er en international internetbaseret encyklopædi, der udgives af den ideelle organisation Wikimedia Foundation [/option]
[/wYh4HfrgvFOJ]
[wwGWJIQF9rr1]
Før en biopsi kan undersøges i mikroskop, skal den behandles kemisk. [/option]
[/wwGWJIQF9rr1]
[xGJ3PFiIgeNa]
Jeg tror, at der kommer mange i år. [/option]
[/xGJ3PFiIgeNa]
[xfjI-IfBFEYJ]
andre venstreorienterede [/option]
[/xfjI-IfBFEYJ]
[y2AQ--Or3hx2]
Der vises film på skolen i morgen. [/option]
[/y2AQ--Or3hx2]
[y4WCxHyTds8G]
Troen på en sådan jordstråling som et sundhedsskadeligt fænomen er særligt udbredt i Skandinavien og tysktalende lande; den har lille udbredelse i USA og de fleste andre lande i verden. Særlig i Tyskland er der blevet bedrevet seriøs forskning omkring fænomenet, men i dag har det ingen plads i den videnskabelige litteratur. [/option]
[/y4WCxHyTds8G]
