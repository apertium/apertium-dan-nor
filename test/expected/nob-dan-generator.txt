[0AQS_LbbTu4t#0] det er en rovfugl, der er udbredt
[/0AQS_LbbTu4t]
[0fNOhzUY_A9i#0] Karsygdomme er en fællesbetegnelse for sygdomme i blodårerne, både arterier og vener.
[/0fNOhzUY_A9i]
[1eDW_0xrRUhX#0] selve by
[/1eDW_0xrRUhX]
[1x4sVzQ8s9s1#0] Om sommeren er det mange dansker, der cykler. Også selv om det regner.
[/1x4sVzQ8s9s1]
[2PNxxSpAa4MU#0] et karakteristisk ravinelandskab
[/2PNxxSpAa4MU]
[3sk50h4xINoB#0] Riget grundlagdes sandsynligvis midt i 800-tallet af svenske vikinger, der underlagde sig området syd for Ladoga, og derefter udvidede herredømmet i 880'erne til også at omfatte Kiev-området.
[/3sk50h4xINoB]
[57KhqMt0wmzr#0] på den anden side
[/57KhqMt0wmzr]
[5pP8Q6Gc8x6t#0] Karsygdomme omfatter sygdomsprocesser som *årebetennelser (arteritis, flebitis), *aterosklerose og blodprop, sjældnere også svulster (*angiom).
[/5pP8Q6Gc8x6t]
[6COebhF1E2Wb#0] *Titanic synkede på jomfrurejsen sin til trods for, at det var erklæret synkefrit
[/6COebhF1E2Wb]
[6KvLKp1kq_xT#0] Det vises film.
[/6KvLKp1kq_xT]
[6w_Cyw_s5ZbW#0] et tysk modsvar
[/6w_Cyw_s5ZbW]
[Bf1sj1J3_fgB#0] den udenrigs- og sikkerhedspolitiske justering
[/Bf1sj1J3_fgB]
[DErPv2VMubRs#0] hvilke byer
[/DErPv2VMubRs]
[E69xPcv3u6jF#0] Jeg synes nu at det er helt *okay. Det er nogle gange hyggeligt at det bare er os fra kontoret.
[/E69xPcv3u6jF]
[FXAuxvkyHAZ2#0] Jeg er ked af det for at det blev dobbeltkonfekt, så jeg gik nok kold og træt af det at have stueetage og ikke første sal
[/FXAuxvkyHAZ2]
[HgPh8_CcygPm#0] et iPhone-problem
[/HgPh8_CcygPm]
[I7NRhBsQ1x4H#0] en lærestol
[/I7NRhBsQ1x4H]
[JOXzDXbN6Lo_#0] en relativt fin lærebog
[/JOXzDXbN6Lo_]
[Jke5Xn5HiMU9#0] Det regner.
[/Jke5Xn5HiMU9]
[LFj9PnMAN0m9#0] Karsygdomme giver derved ophav til nogle af de almindeligste og alvorligste sygdomstilstande som *aorta-aneurisme, slagtilfælde, hjerteinfarkt og *lungeembolisme.
[/LFj9PnMAN0m9]
[LyEBEtFCbGKr#0] hvilken by
[/LyEBEtFCbGKr]
[Mv6OMsSDl14k#0] fredsopgørene
[/Mv6OMsSDl14k]
[NIVpTnBH7NKG#0] en senere omdiskuteret stopordre
[/NIVpTnBH7NKG]
[OTvOC_guDR7P#0] med de samme egenskaber som menneskerne
[/OTvOC_guDR7P]
[PDeZWDuecO4P#0] og hendes bedste ven hedder Tommy
[/PDeZWDuecO4P]
[PgQbI5sKvvOj#0] Ja, det gør jeg helt sikkert, men det er virkeligt surt at kærester ikke får blive med.
[/PgQbI5sKvvOj]
[QJzHLHp6zIUp#0] de allierede
[/QJzHLHp6zIUp]
[RMBRkDyMsyJJ#0] personerne blev dræbte umiddelbart
[/RMBRkDyMsyJJ]
[RyVFEsQt_nVl#0] hun kan ikke lide dette noget særligt
[/RyVFEsQt_nVl]
[T6rtDLF3hR86#0] Det gør ondt.
[/T6rtDLF3hR86]
[U-lthZ4hNtMU#0] i såkaldt fri og grundlæggende forskning
[/U-lthZ4hNtMU]
[UE8i8Dh7zTWM#0] Jeg har åle i min luftpudebåd
[/UE8i8Dh7zTWM]
[VoUl2ZxdUTrn#0] på samme tid
[/VoUl2ZxdUTrn]
[XrGqkvRZ6txF#0] den humanistiske klasse
[/XrGqkvRZ6txF]
[ZjlT_hks49HT#0] tyske styrker
[/ZjlT_hks49HT]
[_2zJOIiBGsSB#0] en senere omdiskuteret ordre
[/_2zJOIiBGsSB]
[_92F_iHPtNNU#0] blev dræbt umiddelbart
[/_92F_iHPtNNU]
[_TUs_iEeAdzD#0] til VG kl. 19
[/_TUs_iEeAdzD]
[a_DwkLG4H_di#0] Det er nu på fredag det er sommerfest
[/a_DwkLG4H_di]
[bExjU9XD6xCU#0] ca. 400 e.Kr.
[/bExjU9XD6xCU]
[bLeZbeB0uX5J#0] den uafhængige stat
[/bLeZbeB0uX5J]
[bZOK2GBR97Z_#0] Stortinget bestemte.
[/bZOK2GBR97Z_]
[dFeFxywBQXR7#0] tyve tredive fyrre halvtreds tres halvfjerds firs nit hundrede
[/dFeFxywBQXR7]
[dKxf-_irk_qd#0] Det starter et nyt kursus om 14 dage.
[/dKxf-_irk_qd]
[e7whSWefDp2f#0] den direkte pressestøtte
[/e7whSWefDp2f]
[e__pH5O_IXrG#0] Det banker på døren. Hvem tror du at det er?
[/e__pH5O_IXrG]
[i9Gb-OzPrtqd#0] største væbnede opstanden
[/i9Gb-OzPrtqd]
[lI7hQm9NW8av#0] Det er dumt, at køre for stærkt.
[/lI7hQm9NW8av]
[liZAWyCanT6V#0] Karsygdomme kan medføre både udvidelser (*aneurismer, åreknuder) og forsnævringer og tilstopninger af blodårerne.
[/liZAWyCanT6V]
[lmZbhKy_sW8Q#0] den nye colombianske stat
[/lmZbhKy_sW8Q]
[lpaouKQGB1Fz#0] Nu skal du tage opvasken
[/lpaouKQGB1Fz]
[pyo8SJjgCh1d#0] Gardariket er i nordisk sagalitteratur navnet på Kievriget, et rige i det nuværende Rusland og Ukraine, der lå omkring Novgorod (på norrønt *Holmgarðr) og Kiev.
[/pyo8SJjgCh1d]
[qrKYmCc0xbAN#0] en kommende ordre
[/qrKYmCc0xbAN]
[soZfxNO1iCxq#0] Der er nogen som banker på døren.
[/soZfxNO1iCxq]
[tNaIcvfkZQ88#0] Jeg tror det kommer mange i år.
[/tNaIcvfkZQ88]
[unXNNvEy3DVm#0] en manglende vilje
[/unXNNvEy3DVm]
[wBwWxw5blUow#0] En tysk-sovjetisk *ukjentordher.
[/wBwWxw5blUow]
[wMR3JZqZ2bQp#0] I praksis har de retslærde
[/wMR3JZqZ2bQp]
[wYh4HfrgvFOJ#0] Wikipedia er en international internetbaseret encyklopædi, der udgives af den ideelle organisation Wikimedia Foundation
[/wYh4HfrgvFOJ]
[xGJ3PFiIgeNa#0] Jeg tror at det kommer mange i år.
[/xGJ3PFiIgeNa]
[xfjI-IfBFEYJ#0] andre venstreorienterede
[/xfjI-IfBFEYJ]
[y2AQ--Or3hx2#0] Det vises film på skolen i morgen.
[/y2AQ--Or3hx2]
